packages/components/src/Button/_docs/Button.stories.tsx

```diff
@@ -165,7 +170,7 @@ export const Loading: StoryFn = () => <LoadingInput isAnimated width={13} />
 export const NativeFormButton: StoryFn = () => (
   <>
     <form className="mb-6" id="unique-form-id">
-      <input type="text" defaultValue="content" />
+      <TextField labelText="Sample text field" defaultValue="content" />
```

Comment:
Given this is a story for Button, I'd actually say it's better not to import the `draft-form` package because it creates more irrelevant maintenance (especially since it's not even in the same repo anymore).

Open to other perspectives though :)

Reply:
It's coming soon™, but also if it's a `devDep`, should be fine. Just need to replace it later.

Reply:
It was already in deps, I didn't have to add it.

This text field & story does seem pointless though so I'd be in favour of just removing it if others are

Reply:
Yeah kill it 🔥

Reply:
I just realised that the text field is linked up with the button, so it's not actually as bad as I thought